### PR TITLE
build: add npm login for different environment test

### PR DESCRIPTION
**Pull request by ryanoolala**
fix pipeline for "Test Different Development Environments" needing to install private npm module from our organisation

### CODE REVIEW

**Review by dalsontws — APPROVED** [review]
LGTM!

File: .github/workflows/test_development_environment.yml [hunk 1]
```diff
@@ -30,13 +30,18 @@ jobs:
...
-            ${{ runner.os }}-build-	
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
             ${{ runner.os }}-
       - name: Setup NPM Token
         run: npm config set //npm.pkg.github.com/:_authToken ${{ secrets.GH_TOKEN_SUPPLYALLY_BOT }}
       - name: Install Packages
-        run: npm ci
+        run: |
+          cd "${{ env.PR_NUMBER }}/v2"
```
Comment (dalsontws) [inline .github/workflows/test_development_environment.yml:40]: oops dont think we need to `cd v2` here!

**Review by dalsontws — CHANGES_REQUESTED** [review]

File: .github/workflows/test_development_environment.yml (hunk 1 above)
Comment (ryanoolala) [reply .github/workflows/test_development_environment.yml:40]: oops

Comment (ryanoolala) [reply .github/workflows/test_development_environment.yml]: good catch, fixed

**Pull request closed without merging**